### PR TITLE
/vg/station BetterTTV/JTV emoji

### PR DESCRIPTION
**Pull request by tigercat2000**
d3athrow/vgstation13/pull/9198

This adds twitch emotes to OOC Chat!
https://twitchemotes.com/

Only works in OOC (technically anything that has the .ooc class but shh)

### CODE REVIEW

**Comment by PJB3005** [comment]
Never thought I'd see the day where twitch chat could come in handy.

Could probably use the twitch chat code for this instead to make it better. 

https://github.com/d3athrow/vgstation13/pull/9198/files

**Comment by ExitGame** [comment]
Remove :faggot: for _obvious reasons_ and we're good.

**Comment by tigercat2000** [comment]
@ExitGame

**Comment by tigercat2000** [comment]
Actually! DNM! Going to remake the entire PR with something different :eyes:

**Comment by tigercat2000** [comment]
Kappa

**Comment by Krausus** [comment]
```
/* Base-64 encoded emoticons below are property of Twitch Interactive and their respective owners */
```
A bunch of copyrighted icons, mostly owned by Twitch, in our codebase, without permission? What could possibly go wrong?

**Comment by tigercat2000** [comment]
@Krausus want me to go bother twitch support?

**Comment by Krausus** [comment]
> Actually, they aren't in our codebase.

Then what's with all the base64-encoded images in `twitchify.css`?

**Comment by tigercat2000** [comment]
the better question is what the cdn url is for, because you're entirely right 😐

File: goon/browserassets/js/browserOutput.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ var opts = {
 	//Options menu
 	'subOptionsLoop': null, //Contains the interval loop for closing the options menu
 	'suppressOptionsClose': false, //Whether or not we should be hiding the suboptions menu
+	'enableTwitchify': true, // I am so sorry
```
Comment (Iamgoofball) [inline goon/browserassets/js/browserOutput.js]: I hate you

**Comment by FreeStylaLT** [comment]
SeemsKappa

**Comment by FlattestGuitar** [comment]
Not Twitch emotes. Just no, please.

For the sake of keeping the memes continuous we should just have all the Twitter and Unicode 9 emojis, like Discord. 

*Please no.*

Also add :erp: . 👌 💯 💦 😩  k thx

**Comment by KasparoVy** [comment]
I don't see why we can't have both.

**Comment by Fox-McCloud** [comment]
After discussing this over, we've agreed that if we're going to have emojis, we should stick with the unicode ones over utilizing Twitch's.

**Pull request closed without merging**